packages/strapi-plugin-users-permissions/config/schema.graphql.js

```diff
@@ -53,18 +53,18 @@ module.exports = {
       user: UsersPermissionsMe!
     }
 
-    type ForgotPassword {
-      ok: Boolean
+    type UserPersmissionsPasswordPayload {
+      ok: Boolean!
     }
   `,
   query: `
     me: UsersPermissionsMe
   `,
   mutation: `
     login(input: UsersPermissionsLoginInput!): UsersPermissionsLoginPayload!
-    register(input: UsersPermissionsRegisterInput!): UsersPermissionsLoginPayload!
-    forgotPassword(email: String!): ForgotPassword
-    changePassword(password: String!, passwordConfirmation: String!, code: String!): UsersPermissionsLoginPayload
+    register(input: UserInput!): UsersPermissionsLoginPayload!
```

Comment:
Small mistake when you fixed the conflicts :) We should keep the UsersPermissionsRegisterInput :)

Reply:
Hi @ScottAgirs  will you be able to make the change ?

Reply:
@alexandrebodin hey Alex, thanks for the follow-up, my apologies, I must have missed the notification about your review. 

Just pushed 🚀